### ISSUE TITLE
.ruby-version  doesn't match the documentation

### ISSUE THREAD

**Issue by Wenzel** [jedi4ever/veewee#1043]
Hi
your `.ruby-version` ask to use [`ruby-1.9.3-p547`](https://github.com/jedi4ever/veewee/blob/master/.ruby-version)

But at the same time, in your documentation your have written **IMPORTANT: Currently supported versions of ruby: 2.2.0**

Did you forget to update the `.ruby-version` file ?

**Comment by mpapis** [comment]
the documentation is to strict, all ruby versions should work but the current ruby version is `2.2.1` ... can you open a pull request to change the .ruby-version to `2.2.1` and the docs:
```md
Currently supported versions of ruby: 1.9.3+
IMPORTANT: for best results always use latest ruby
```

**Comment by mpapis** [comment]
fixed in  #1045
